nbs/examples/ElectricityPeakForecasting.ipynb

```diff
@@ -0,0 +1,14449 @@
+{
```

Comment:
<p>Good oportunity to quote your great Nhits paper :)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/statsforecast/pull/335/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='37'/>

Reply:
<p>Done!</p>